src/Adapter/Module/ModuleDataProvider.php

```diff
@@ -144,7 +144,7 @@ public function getInstalled(): array
 
         $id_shops = (new Context())->getContextListShopID();
         if (count($id_shops) > 0) {
-            $select .= ', ms.`id_module` as active';
+            $select .= ', ms.`id_module` as active, ms.`enable_device` as active_on_mobile';
```

Comment:
@mflasquin Enabling and disabling modules on mobiles have been removed from v9 on purpose, don't add this code back. :-)

Reply:
@Hlavtox @mflasquin we must be careful how we merge/modify this then Because the `enable_device` field value may not be important anymore however the module_shop row is important for multishop module's association

And we recently fixed a bug related to this piece of code in those PRs https://github.com/PrestaShop/PrestaShop/pull/32949 and https://github.com/PrestaShop/PrestaShop/pull/31379 but for some reason this important modification is not on develop and not in this PR There's something wrong here

Reply:
Fixed with this PR too : https://github.com/PrestaShop/PrestaShop/pull/31151

Reply:
Can you check updates @Hlavtox @jolelievre ?